src/backend/distributed/master/master_cache_table_shards.c

```diff
@@ -0,0 +1,191 @@
+/*-------------------------------------------------------------------------
+ *
+ * master_cache_table_shards.c
```

Comment:
This header comment is inaccurate, and I think the file name should be changed, too, no?